src/Form/Form.cpp

```diff
@@ -391,6 +392,13 @@ WndForm::ShowModal()
       }
 #endif
 
+#ifdef ANDROID
+      if (event.GetKeyCode() == KEY_ESCAPE || event.GetKeyCode() == KEYCODE_BUTTON_R1) {
```

Comment:
The `ESCAPE` check is redundant. The `BUTTON_R1` check is not explained.

Reply:
This is another place where the BUTTON_L1/BUTTON_R1 change needs to be made so that the controller and 'escape' from the menu.

Reply:
Would it be a good idea for these to be commented in each location?

Reply:
My brain is unable to parse your replies.

Reply:
i'll use the Bridge method to map all inputs so this can be removed.